scripts/user-actions/remove-number.js

```diff
@@ -21,6 +21,7 @@ const removeNumber = (arrOfNumbers, toRemove) => {
   }
 
   // write the logic!
```

Comment:
I commented my opinion about the logic. So, you can test it once. I hope it would help.

Reply:
I checked on js-tutor, it worked 👍

Reply:
thank you, guys!